### DISCUSSION TITLE
Check recorder status

### DISCUSSION THREAD

**Discussion by Faaatman** [Canardoux/flutter_sound#744] [Q&A] [answered]
Is there a way to check if another application is using the microphone when opening a session or when starting the recorder?
Thanks!

**Comment by Larpoux** [comment, marked as answer]
No. I do not know how to do that, neither on ios nor on android.
All the App run in sandboxes and it is not simple to have access on resources used by those App.

**Reply by Faaatman** [reply]
Perfect. thanks!
